pkgs/by-name/ta/tabby-terminal/package.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchurl, pkgs, lib, alsa-lib }:
+
+stdenv.mkDerivation rec {
+  name = "tabby-terminal";
+  version = "1.0.207";
+  src = fetchurl {
+    url = "https://github.com/Eugeny/tabby/releases/download/v${version}/tabby-${version}-linux-x64.deb";
+    hash = "sha256-PTTkL3+mYb7KM8fDUmgCuAF2lU88fYOstGWp/O5WZas=";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    dpkg
+    makeWrapper
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    dpkg -x $src .
+
+    runHook postUnpack
+  '';
+
+  buildInputs = with pkgs; [
+    glib
+    nss
+    nspr
+    atk
+    cups
+    dbus
+    libdrm
+    gtk3
+    pango
+    cairo
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    libxkbcommon
+    mesa
+    expat
+    xorg.libxcb
+    alsa-lib
+    libGL
+    libsecret
+    musl
+  ];
+
+  postInstall = ''
+    mkdir -p $out/{opt,bin}
+    cp -r opt $out
+    wrapProgram "$out/opt/Tabby/tabby" --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland --enable-features=WaylandWindowDecorations}} --no-sandbox" --set LD_LIBRARY_PATH=${lib.makeLibraryPath buildInputs}
+    cp -r usr/* $out
+
+    ln -s $out/opt/Tabby/tabby $out/bin/tabby
+
+    substituteInPlace $out/share/applications/tabby.desktop \
+      --replace "/opt" $out/opt
```

Comment:
Be careful with absolute path in desktop entry. See https://github.com/NixOS/nixpkgs/issues/308324

Reply:
Fixed

Reply:
doesn't seem to be fixed:
```
$ cat result/share/applications/tabby.desktop | grep Exec
Exec=/nix/store/5j7s1cl4y7f6v57iyd998rm1dg828hym-tabby-terminal-1.0.207/opt/Tabby/tabby --no-sandbox %U
```